docs/reference/query-languages/esql/kibana/definition/commands/inline_stats.json

```diff
@@ -0,0 +1,5 @@
+{
+  "comment" : "This is generated by ESQL’s DocsV3Support. Do not edit it. See ../README.md for how to regenerate it.",
+  "type" : "command",
+  "name" : "inline_stats"
```

Comment:
Thought: Huh, interesting. We also have commands that have proper underscores in them, I wonder if that's an issue with our docs generation.

Reply:
I assume not, [lookup_join](https://github.com/elastic/elasticsearch/blob/main/docs/reference/query-languages/esql/kibana/definition/commands/lookup_join.json) didn't trigger any yet.